packages/plugins/content/slate/src/plugins/Plugin.ts

```diff
@@ -49,12 +67,12 @@ export default class Plugin {
    */
   public schema: {
     // tslint:disable-next-line:no-any
-    document?: { [key: string]: any };
+    document?: { [key: string]: any }
```

Comment:
ok these are the more annyoing formatting changes. Strange that the semicolons got lost here. i try to figure out why.

Reply:
ok i added a .prettierrc file to force semicolons, not sure why in some files it kept semicolons and in others not.